tests/network/test_fail_request.py

```diff
@@ -476,6 +476,7 @@ async def test_fail_request_completes_new_request_still_blocks(
             "params": {
                 "url": example_url,
                 "context": context_id,
+                "wait": "complete",
```

Comment:
Can we use the helper `goto_url` that way we do the same default everywhere.

Reply:
Oops, I think we can't do that as that waits for the command to finish, should we allow the helper to not wait for the command? and use it in all instances of the the `browsingContext.navigate`?

Reply:
`goto_url` does not work here because it skips too many events

Reply:
I don't oppose your suggestion but it will require a lot of refactoring, and it's out of scope of the intent of this PR. Maybe in a separate one